README.md

```diff
@@ -1,24 +1,76 @@
+Data collected is used to evaluate the success of the tools. Then, the tools will be trained on larger batches of data to automatically retrieve information important to the archival process.
 
-Each directory in this repository contains a dataset from a specific annotation project. The directory name is the name of the project. Each directory contains the following files:
+## Structure of This Repository/Directory
+ - `batches` subdirectory (1)
+ - project subdirectories (5)
+ - this repository's `readme.md`
 
-* `README.md`: annotation project-specific information, such as
+### Batches Subdirectory 
+The first directory is the special `batches` directory. This special directory maintains tracking information for the whole repository/annotation effort -  
+Batches are the identity tags of a group of media/GUID annotated for a project-found-below in one go. 
+There are possibly multiple times/events where data is annotated/created for this dataset. Each event is a batch.
+Specifically, this directory contains `.txt` files named after the batch name. Batches are usually named after their relevant GitHub issue from [AAPB-CLAMS collaboration repository](https://github.com/clamsproject/aapb-collaboration).  
+  * Each line in a `.txt` file _must_ be a single AAPB GUID, with an exception of any lines starts with `#` which denotes a comment.  
+  * A GUID is a unique identifying string that can be used at their website to find one particular media and its supporting files, eg. `cpb-aacip-96d289b264c` at https://americanarchive.org/catalog/cpb-aacip-96d289b264c.
+
+### Project Subdirectories 
+Each directory in this repository represents a specific annotation project, its datasets and processing tools.    
+This includes its `raw annotated data file`, `software-suite for converting from raw to gold`, `gold-formatted-final-output data file for tool ingestion`,
+and `that project's readme.md explaining it plus its annotation guidelines`.  
+The directory name is the name of the project. Each directory contains the following files:
+
+* **RAW INTERMEDIATE DATASET FILES**: `YYMMDD-batchName` directory - these sub-directories contains raw output files from the manual annotation process created by the annotation tool (or by hand like a `.csv` file). 
+Different annotation tools create different file formats with diverse formatting. 
+  * The `YYMMDD-` prefix _must_ indicate the time when a batch of annotation is conducted. 
+The `batchName` part of the directory name _must_ match only one of `.txt` files in the batches. 
+* **CODEBASE FOR CONVERSION**: `process.{sh,py}` - a script to process the raw manual annotations and generate the publically-available "gold" dataset. In addition to this file, if the code requires additional dependencies/scripts, they can stay in the same directory. In case the dependencies are managed by a package manager (e.g., `pip`), the project manager _must_ provide relevant information in the `README.md` file.
```

Comment:
"`README.md` file or package manager specific specification file (e.g., `requirements.txt` for `pip`)" ?

Reply:
What do you want to say here?

Reply:
I meant the PM can provide the dependency information in the readme, but also can do by having a machine-friendly file with the list of dependencies.